http/src/main/resources/reference.conf

```diff
@@ -61,7 +61,7 @@ dataproc {
     numberOfPreemptibleWorkers = 0
     region = "us-central1"
   }
-  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/test-1-4-51-debian10-qi-2021-07-21-16-54-07"
+  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/test-1-4-51-debian10-qi-2021-07-22-11-27-42"
```

Comment:
Why is the naming scheme different? They used to start with `leonardo-custom-image` or something like that.

Reply:
these images are still generated locally

Reply:
We could still name them the same way as before. It's sometimes helpful to be able to filter for images with a known simple prefix.

Reply:
hmm..I've seen various prefixes or postfixes..but yea, I can I see a consistent prefix can be helpful...altho originally I wanted to differentiate locally generated images from jenkins generated ones

Reply:
Is the jenkins job still not working?

Reply:
nope...but it seems things are better after merging the previous PR..see https://fc-jenkins.dsp-techops.broadinstitute.org/view/Leonardo/job/leonardo-custom-image/194/